### ISSUE TITLE
fix: FileSystemException on mason get

### ISSUE THREAD

**Issue by Prn-Ice** [felangel/mason#828]
**Description**

Whenever I run mason get in my project directory, I get this error.
```sh
FileSystemException: Rename failed, path = '/tmp/IPHPMH' (OS Error: Invalid cross-device link, errno = 18)
```

**Steps To Reproduce**

1. run mason init
2. add this to your mason.yaml
```yaml
bricks:
  hello: 0.1.0+1
  annoying_analysis_options: ^0.0.1
  feature_brick: ^0.4.0
  app_ui: ^0.0.4
  heavy_repository_package:
    git:
      url: https://github.com/Prn-Ice/mason_bricks.git
      path: bricks/heavy_repository_package
  heavy_service_package: ^0.0.2
```
4. run mason get
5. See error

**Expected Behavior**

No error.

**Screenshots**

**Additional Context**

Flutter version: 
```sh
Flutter 3.7.7 • channel stable • https://github.com/flutter/flutter.git
Framework • revision 2ad6cd72c0 (2 weeks ago) • 2023-03-08 09:41:59 -0800
Engine • revision 1837b5be5f
Tools • Dart 2.19.4 • DevTools 2.20.1
```

OS: Endeavour OS

**Comment by felangel** [comment]
Hi @Prn-Ice 👋 
Thanks for opening an issue!

Can you also please provide the version of mason_cli you're using (`mason --version`)? Thanks!

**Comment by Prn-Ice** [comment]
0.1.0-dev.50

**Comment by felangel** [comment]
> 0.1.0-dev.50

Thanks I'll have a look later today and report back, sorry for the inconvenience!

**Comment by felangel** [comment]
@Prn-Ice this has been fixed in mason v0.1.0-dev.48. You should be able to re-activate `mason_cli` to pull the fix:
`dart pub global activate mason_cli`

Let me know if you're still facing issues after re-activating, thanks!

**Comment by Prn-Ice** [comment]
Works as expected now, thank you.
